utils/generate-docs/Schema/SchemaNode/BackwardsCompatibleObjectSchemaNode.ts

```diff
@@ -41,9 +40,6 @@ export default class BackwardsCompatibleObjectSchemaNode extends SchemaNode {
 
   protected basename = () => path.basename(this.id(), ".schema.json");
 
-  protected directory = () =>
```

Comment:
This is what's creating the undesired path that is being written to during document generation for the Compatibility Wrappers.